modules/alb-logs-to-s3-to-cloudwatch/bucket.tf

```diff
@@ -14,6 +14,32 @@ resource "aws_s3_bucket" "bucket" {
   count = var.create_alb_log_bucket ? 1 : 0
 
   bucket = var.alb_log_bucket_name
+  versioning {
+    enabled = var.enabled
+  }
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm = var.sse_algorithm
+      }
+    }
+  }
+  dynamic "logging" {
+    for_each = var.logging
+    content {
+      target_bucket = logging.value["target_bucket"]
+      target_prefix = "log/${var.alb_log_bucket_name}"
+    }
+  }
+}
+
+resource "aws_s3_bucket_public_access_block" "access" {
```

Comment:
Let the users pass these "true" values dynamically and not have it hardcoded here. Also, you can set "true" the default values for those variables.